api-reference/10 UI Components/Errors and Warnings/W1018.md

```diff
@@ -3,7 +3,7 @@ id: ErrorsUIWidgets.W1018
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Occurs in the DataGrid UI component when infinite scrolling is used with multiple selection.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Set **selection**.[deferred](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#deferred) to **true** or set **selection**.[selectAllMode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#selectAllMode) to *"page"*.
```

Comment:
It looks like we mention four totally unconnected things:

- Infinite scrolling
- Multiple selection
- Deferred selection
- Selection of all records

It doesn't add up to me.